# HG changeset patch
# User Andrew John Hughes <gnu.andrew@redhat.com>
# Date 1390502807 0
#      Thu Jan 23 18:46:47 2014 +0000
# Node ID fa97228fa38479b74e413bc1c2d2aeef0057b6dc
# Parent  f49a6db8092f35b9f12af481c81ffb2aab8c3fdd
Remove unneeded jdk_generic_profile.sh patch that breaks when system LCMS is disabled.

2014-01-23  Andrew John Hughes  <gnu.andrew@redhat.com>

	* patches/alternative_krb5_cache_fixup.patch:
	Remove unneeded jdk_generic_profile.sh fragment
	that doesn't apply when system LCMS is disabled.

diff -r f49a6db8092f -r fa97228fa384 ChangeLog
--- a/ChangeLog	Thu Jan 23 00:53:36 2014 +0000
+++ b/ChangeLog	Thu Jan 23 18:46:47 2014 +0000
@@ -1,3 +1,9 @@
+2014-01-23  Andrew John Hughes  <gnu.andrew@redhat.com>
+
+	* patches/alternative_krb5_cache_fixup.patch:
+	Remove unneeded jdk_generic_profile.sh fragment
+	that doesn't apply when system LCMS is disabled.
+
 2014-01-23  Andrew John Hughes  <gnu.andrew@redhat.com>
 
 	* configure.ac: Set to 1.13.2pre.
diff -r f49a6db8092f -r fa97228fa384 patches/alternative_krb5_cache_fixup.patch
--- a/patches/alternative_krb5_cache_fixup.patch	Thu Jan 23 00:53:36 2014 +0000
+++ b/patches/alternative_krb5_cache_fixup.patch	Thu Jan 23 18:46:47 2014 +0000
@@ -1,21 +1,3 @@
-diff -Nru openjdk.orig/jdk/make/jdk_generic_profile.sh openjdk/jdk/make/jdk_generic_profile.sh
---- openjdk.orig/jdk/make/jdk_generic_profile.sh	2013-08-30 02:41:17.276807687 +0100
-+++ openjdk/jdk/make/jdk_generic_profile.sh	2013-08-30 12:11:12.428923185 +0100
-@@ -467,5 +467,14 @@
- export LCMS_CFLAGS
- export LCMS_LIBS
- 
-+# Export variables for system krb5
-+# KRB5_CFLAGS and KRB5_LIBS tell the compiler how to compile and
-+# link against Kerberos
-+if [ "${KRB5_LIBS}" = "" ] ; then
-+    KRB5_LIBS="-lkrb5"
-+fi
-+export KRB5_LIBS
-+
- # IcedTea defaults; use system libraries
- export SYSTEM_LCMS=true
-+export SYSTEM_KRB5=true
 diff -Nru openjdk.orig/jdk/make/sun/security/krb5/internal/ccache/Makefile openjdk/jdk/make/sun/security/krb5/internal/ccache/Makefile
 --- openjdk.orig/jdk/make/sun/security/krb5/internal/ccache/Makefile	2013-08-30 02:41:17.408809749 +0100
 +++ openjdk/jdk/make/sun/security/krb5/internal/ccache/Makefile	2013-08-30 12:10:34.336330523 +0100
